source/features/hide-inactive-deployments.tsx

```diff
@@ -2,11 +2,14 @@ import select from 'select-dom';
 import features from '../libs/features';
 
 function init(): void {
-	const deployments = select.all('.TimelineItem .deployment-meta');
+	const deployments = select.all(`
+		.TimelineItem [title^="Deployment Status Label:"],
+		.js-socket-channel[data-url$="/pull_requests/events/deployed"] .TimelineItem a.btn
+	`);
```

Comment:
Actually I think this is the only change required in this file, I forgot that we always select _all_ the deployments so that we can leave the last one on the page:

```suggestion
	// Selects all the deployments first so that we can leave the last one on the page
	const deployments = select.all('.js-socket-channel[data-url$="/pull_requests/events/deployed"']');
```